composite/proof-of-reserves/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('../../.eslintrc.ts.js'),
+}
```

Comment:
```suggestion
module.exports = require('../../.eslintrc.ts.js')
```

Reply:
Doesnt eslint resolve by crawling up the directory tree anyway?

Reply:
I think we do, as we have on for JS `.eslintrc.js` and the other for TS `.eslintrc.ts.js`